packages/http/src/resources/Hotspots.ts

```diff
@@ -60,4 +60,14 @@ export default class Hotspots {
     } = await this.client.get(url)
     return new Hotspot(this.client, hotspot)
   }
+
+  async elected(block?: number): Promise<ResourceList<Hotspot>> {
```

Comment:
why is this present in this PR?

Reply:
woops, forked from the last branch I was working on instead of from master